internal/db/seeds/emed.go

```diff
@@ -0,0 +1,6 @@
+package seeds
+
+import "embed"
+
+//go:embed *.sql
+var EmbeddedSeeds embed.FS
```

Comment:
Had to move both migrations and seeds into the directory because the way the Tern API is designed today, it does not allow callers to locate files from subdirectories.